### ISSUE TITLE
"Prevent skipping forward in a video" disappears

### ISSUE THREAD

**Issue by RogueAgent991** [h5p/moodle-mod_hvp#512]
Moodle 3.11

To reproduce:
- in an H5P module, click "Prevent skipping forward in a video", then save the module
- go back into the module, and "Prevent skipping forward in a video" no longer shows

Thx

**Comment by otacke** [comment]
@RogueAgent991 Hi! I am Oliver and I was made a community maintainer for H5P Group's repositories a while back, and I am still in the process trying to tackle all the open issues and pull requests. Just to make that clear, I am not employed by H5P Group, I merely invest some of my spare time to help the H5P project.

I am terribly sorry that you have not received a reply before.

Could you please let me know whether this is still an issue? The ticket is old :-/

**Comment by RogueAgent991** [comment]
@otacke 
You are correct -- this is a bit old. I no longer have this issue, but cannot remember the resolution.
